### PR TITLE
Update Cronos Play C++ SDK and align with v0.0.2-alpha

### PR DESCRIPTION
**Pull request by damoncro**
Solutions:
1. Update Cronos Play C++ SDK and align with v0.0.2-alpha
2. Add Examples section fro cronos play cpp sdk

### CODE REVIEW

File: cronos-play/getting-started_cpp.md [hunk 1]
```diff
@@ -9,170 +9,131 @@
...
 
-1. Link `play_cpp_sdk` static library in your build flow (check `demo/Makefile` for more details)
+    ```
+    git clone https://github.com/crypto-com/play-cpp-sdk.git
+    ```
+2. Unzip the archive file into `demo` folder, and replace the original `sdk` folder
+3.  Under `demo` folder and build the `demo` project
 
-Windows
+    ```
```
Comment (zkaizhi) [inline cronos-play/getting-started_cpp.md:59]: small detail, could you add ```bash as the syntax highlighting, it'll be much nicer on gitbook

Comment (damoncro) [reply cronos-play/getting-started_cpp.md]: Updated.

File: cronos-play/getting-started_cpp.md [hunk 2]
```diff
@@ -9,175 +9,137 @@
...
-#### Mac
+### Mac
 
-1. Run `make build_cpp` or `make cppx86_64`
-2. Run `make install`, libraries and bindings will be copied into a new created folder: `build`
+1. Run `make`
+2. Run `make install`, libraries and bindings will be copied into a new created folder: `install`
 
-#### Linux
+### Linux
```
Comment (damoncro) [inline cronos-play/getting-started_cpp.md:129]: Gitbook seems can not support level 4 heading..

File: cronos-play/getting-started_cpp.md [hunk 3]
```diff
@@ -9,175 +9,137 @@
...
-1. Link `play_cpp_sdk` static library in your build flow (check `demo/Makefile` for more details)
-
-Windows
-
-```powershell
-lib\play_cpp_sdk.lib
-```
-
-Mac or Linux
+* Link the `play_cpp_sdk` static or dynamic library, `cxxbridge1` static library, and sources (\*.cc) into your build system (Visual Studio solution, CMake or Makefile). For more details, check out [Cronos Play Docs](https://github.com/crypto-org-chain/cronos-play-docs).
```
Comment (damoncro) [inline cronos-play/getting-started_cpp.md:108]: Will cronos-play-docs be abandoned?

File: cronos-play/getting-started_cpp.md [hunk 4]
```diff
@@ -410,3 +372,8 @@ function `safe_batch_transfer_from` - Batched version of `safeTransferFrom`.
     const ::org::defi_wallet_core::PrivateKey & private_key
 ) const
 ```
+
+## Cronos Play C++ API Reference
+
+API Reference can be found [here](https://leslie-h-cheung.gitbook.io/cronos-play-c++-sdk/readme-1)
```
Comment (zkaizhi) [inline cronos-play/getting-started_cpp.md:378]: Is there a public link for this one?

Comment (damoncro) [reply cronos-play/getting-started_cpp.md:378]: Yes, for this moment.

File: cronos-play/getting-started_cpp.md (hunk 3 above)
Comment (damoncro) [reply cronos-play/getting-started_cpp.md]: Updated to https://leslie-h-cheung.gitbook.io/cronos-play-c++-sdk/

**Pull request merged**